src/pilot/config_idrac.py

```diff
@@ -541,6 +541,22 @@ def reset_idrac(drac_client, ip_service_tag):
     drac_client.reset_idrac(wait=True)
 
 
+def get_lifecycle_out_of_recovery(drac_client, ip_service_tag):
```

Comment:
Would you mind changing "get_lifecycle_out_of_recovery" to "take_lifecycle_out_of_recovery"?

Reply:
I have changed the method name to "take_lifecycle_out_of_recovery".